app/controlplane/internal/service/signing.go

```diff
@@ -53,3 +53,15 @@ func (s *SigningService) GenerateSigningCert(ctx context.Context, req *v1.Genera
 
 	return &v1.GenerateSigningCertResponse{Chain: &v1.CertificateChain{Certificates: certs}}, nil
 }
+
+func (s *SigningService) GetTrustedRoot(ctx context.Context, _ *v1.GetTrustedRootRequest) (*v1.GetTrustedRootResponse, error) {
```

Comment:
just to make sure, what happens if you are not using keyless att all?

Reply:
Good point. We can choose to either return a NotImplementedErr or just an empty list. I would vote for the error, just to denote that it's not configured.

Reply:
Update: I'm returning an error, as we were doing in `GetSigningCert` method as well.